### PR TITLE
add icc and imsi to match updated responses

### PR DESCRIPTION
**Pull request by coreypurcell**
## Ticket

https://jira.bqsoft.com/browse/ATOM-6540

## What this does

add icc and imsi to match updated responses.

## Why we did this

Telispire doesn't show them in the docs but they are in the WSDL. I need them for changing the device, so that I know if the change iccid succeeded.

## Overall, I feel this way about this code

I FEEL HAPPY! I FEEL HAPPY!
https://www.youtube.com/watch?v=grbSQ6O6kbs

### CODE REVIEW

File: lib/conduit/telispire/parsers/line_attributes.rb [hunk 1]
```diff
@@ -90,6 +90,14 @@ def self.included(receiver)
       receiver.attribute :order_id do
         content_for "//Orderid"
       end
+
+      receiver.attribute :icc do
```
Comment (pedromartinez) [inline lib/conduit/telispire/parsers/line_attributes.rb]: We usually call this iccid in Connect.

Comment (coreypurcell) [inline lib/conduit/telispire/parsers/line_attributes.rb]: yes we do. I'm OK with either, translate it here, or we leave it as telispire defined it.

File: lib/conduit/telispire/parsers/line_attributes.rb [hunk 2]
```diff
@@ -91,7 +91,7 @@ def self.included(receiver)
         content_for "//Orderid"
       end
 
-      receiver.attribute :icc do
+      receiver.attribute :icc_id do
```
Comment (pedromartinez) [inline lib/conduit/telispire/parsers/line_attributes.rb]: Hmm, I think I'd prefer iccid for everything as a standard. It just eliminates variables.

Comment (coreypurcell) [inline lib/conduit/telispire/parsers/line_attributes.rb]: done

**Pull request merged**